meta-mender-core/recipes-bsp/u-boot/patches/0002-Generic-boot-code-for-Mender.patch

```diff
@@ -143,8 +143,15 @@ index 0000000000..55f9bca7a1
 +    "run bootcmd\0"
 +#endif
 +
++#ifndef CONFIG_LOADADDR
++# define MENDER_DEFAULT_LOADADDR
++#else
++# define MENDER_DEFAULT_LOADADDR "kernel_addr_r=" __stringify(CONFIG_LOADADDR) "\0"
++#endif
```

Comment:
Btw, I'm considering switching to `loadaddr` simply because it seems to be the preferred one in `env_default.h`. `kernel_addr_r` was somewhat arbitrarily chosen.